aspnetcore/web-api/define-controller/samples/WebApiSample.Api.21/Controllers/PetsController.cs

```diff
@@ -19,12 +19,14 @@ public PetsController(PetsRepository repository)
         }
 
         [HttpGet]
+        [ProducesResponseType(typeof(IEnumerable<Pet>), 200)]
```

Comment:
A 200 status code and the return type are implied because `ActionResult<T>` is returned.

```suggestion
```